src/patient/patientActions.jsx

```diff
@@ -12,8 +12,9 @@ const getIC3PatientsFailed = (message) => ({
   message: message
 });
 
-const getIC3PatientScreeningData = (patient) => ({
+const getIC3PatientScreeningData = (patient, formInstanceId) => ({
   patient: patient,
+  formInstanceId: formInstanceId,
```

Comment:
Hmm, now that I’m seeing this I think I get what you mean with this approach being unintuitive. A function comment would help. Also maybe a name like `instanceIdOfFormBeingSubmitted`

Reply:
Thanks @brandones , I added a comment